From e585cea1d27be3769c33cf1c0a38bf0de41e2bf9 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Tue, 22 Mar 2016 16:05:32 +0800
Subject: [PATCH] dependence on correct header file path

To avoid the below building error, use the correct header file path.
xlators/features/glupy/src/glupy.c:13:20: fatal error: Python.h:
No such file or directory

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>

---
 xlators/features/glupy/src/glupy.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xlators/features/glupy/src/glupy.c b/xlators/features/glupy/src/glupy.c
index bca4764..7c2b271 100644
--- a/xlators/features/glupy/src/glupy.c
+++ b/xlators/features/glupy/src/glupy.c
@@ -10,7 +10,7 @@
 
 #include <ctype.h>
 #include <sys/uio.h>
-#include <Python.h>
+#include <python2.7/Python.h>
 
 #include "glusterfs.h"
 #include "xlator.h"
